src/styles/style.css

```diff
@@ -0,0 +1,79 @@
+@import url("../styles/variables.css");
+@import url("../styles/rating.css");
+
```

Comment:
correct me if I'm wrong, but separated css files should be for each block in your css code?